charts/loki-stack/values.yaml

```diff
@@ -13,7 +13,7 @@ grafana:
     datasources:
       enabled: true
   image:
-    tag: 6.7.0
+    tag: 7.4.3
```

Comment:
Is there a reason that we explicitly override the imageTag here? Why we don't us the subchart defaults?